docker/ckan/2.9.9-core.Dockerfile

```diff
@@ -0,0 +1,78 @@
+# See CKAN docs on installation from Docker Compose on usage
```

Comment:
optional: I'd recommend running [Hadolint](https://github.com/hadolint/hadolint#haskell-dockerfile-linter) on this file and considering its suggestions. (It occasionally comes up with false positives but mostly its suggestions are good.)

Reply:
ok, will try it out